core/src/mysql/federation.rs

```diff
@@ -29,9 +31,9 @@ impl MySQLTable {
         let fed_provider = Arc::new(SQLFederationProvider::new(self));
```

Comment:
Instead of using try_from here, RemoteTableRef can be directly created from table reference
```rust
let remote_table_ref = RemoteTableRef::from(self.base_table.table_reference.clone());
```